src/main/kotlin/calculator/Calculator.kt

```diff
@@ -0,0 +1,60 @@
+package calculator
+
+import calculator.Operator.ADDITION
+import calculator.Operator.DIVISION
+import calculator.Operator.MULTIPLICATION
+import calculator.Operator.SUBTRACTION
+
+object Calculator {
+    fun compute(input: String): Int {
+        validateInput(input)
+        val inputList = input.toInputList()
+        return calculate(inputs = inputList)
+    }
+
+    fun validateInput(input: String?): String {
+        if (input.isNullOrBlank()) throw IllegalArgumentException("Input is null or blank")
```

Comment:
```suggestion
        require(!input.isNullOrBlank()) { "Input is null or blank" }
```

Kotlin provides helpful built-in functions like `require` and `check` for argument validation.
They make precondition checks more concise and expressive.

Here’s a reference that might be useful:
- [[Hyperskill] Require, check, and assert](https://hyperskill.org/learn/step/33443)